src/components/sidebar/helpers/generate-resources-nav-items.ts

```diff
@@ -150,12 +151,16 @@ function generateResourcesNavItems(
 			title: 'Support',
 			href: DEFAULT_SUPPORT_LINK,
 		},
-		{
-			title: 'GitHub',
-			href: productSlug
-				? GITHUB_LINKS_BY_PRODUCT_SLUG[productSlug]
-				: DEFAULT_GITHUB_LINK,
-		},
+		...(productSlug !== 'waypoint'
```

Comment:
Not a fan of this as I'd rather the data filtered out before it hits this logic, but we are using the same logic just a few lines up: https://github.com/hashicorp/dev-portal/blob/ed5bff48fa84a63dbbe162a8659a008913a40734/src/components/sidebar/helpers/generate-resources-nav-items.ts#L134 So I'll allow it.

Reply:
The default here would be the HashiCorp org. I asked in the ticket if Tony & Emily are okay with that, as that would be preferable here! Will let you know.

Reply:
🤷🏻‍♂️